### DISCUSSION TITLE
2023/05/08/whining/

### DISCUSSION THREAD

**Discussion by giscus** [FloranceYeh/FloranceYeh.github.io#9] [Announcements]
# 2023/05/08/whining/

It's Florance!

https://floranceyeh.netlify.app/2023/05/08/whining/

**Comment by FloranceYeh** [comment]
|统计               |结果    |
|------------------|--------|
|`单词数`           |335    |
|`句数`             |19     |
|`段数`             |18     |
|`字符数(包含空格)`  |1785   |
|`字符数(不包含空格)`|1434   |
|`阅读时长`          |1分13秒|

> 笑

**Comment by ERA-YES** [comment]
jb

**Reply by FloranceYeh** [reply]
gb
